src/main/java/xogame/board_utils/Generator.java

```diff
@@ -0,0 +1,91 @@
+package xogame.board_utils;
+
+import java.util.Arrays;
+
+import xogame.models.Board;
+import xogame.utils.Result;
+
+public class Generator {
+    public static Result<Board> fill_nth_row_of_the_board_with_symbol(int board_size, int rowNumber, int symbol) {
+        int[] array = new int[board_size * board_size];
+
+        if (rowNumber < board_size) {
+            Arrays.fill(array, board_size * rowNumber, (rowNumber + 1) * board_size, symbol);
+
+            return new Result<Board>(new Board(array));
+        } else {
+            return new Result<Board>(new Board(array), "There is no row like this.");
+        }
+    }
+
+    public static Result<Board> fill_nth_column_of_the_board_with_symbol(int board_size, int columnNumber, int symbol) {
+        int[] array = new int[board_size * board_size];
+        int[] arrayOfIndicies = Generator.generateIndiciesOfTheElementsOfTheColumn(board_size, columnNumber);
```

Comment:
Why do we need the `generateIndiciesOfTheElementsOfTheColumn` here? I cannot see the value it adds to the code.